internal/util/tftemplate.go

```diff
@@ -119,15 +118,13 @@ func NewMainTFTemplate(fs FileSystemer) *MainTFTemplate {
 		/*
 			#nosec CWE-703: failures accounted for in error returned by Write function. TODO: refactor this
 		 */
-		os.Mkdir(tfWorkDir, os.ModeDir|os.FileMode(int(0700)))
+		_ = os.Mkdir(tfWorkDir, os.ModeDir|os.FileMode(int(0700)))
 	}
 
 	tfStateFilePath := fs.GetTerraformStateFile()
 
 	tf := &MainTFTemplate{
-		TFVars:               []TFVar{},
```

Comment:
`TfVars` and `Version` are set in the main command initialization. 
This is an example of where we are setting defaults in too many different places